### PR TITLE
chore: Fix broken links in app README, support .venv convention

### PR DESCRIPTION
**Pull request by cephalization**
Let me know if there are any consequences to having .venv in the project dir that I can't think of

### CODE REVIEW

**Comment by cephalization** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by cephalization** [comment]
recheck

**Review by RogerHYang — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -30,4 +30,7 @@ pyrightconfig.json
 tutorials/internal/demo_llama_index/*.json
 examples/agent_framework_comparison/utils/saved_traces/*.parquet
 .env
+
+# python environments
 .conda
+.venv
```
Comment (axiomofjoy) [inline .gitignore:36]: This seems to already be ignored for me. Not sure how.

Comment (cephalization) [reply .gitignore:36]: Maybe you used `git skip-worktree` [docs](https://git-scm.com/docs/git-update-index#Documentation/git-update-index.txt---no-skip-worktree) or a global git ignore file.

I'm down to do either if we feel this isn't necessary

Comment (RogerHYang) [reply .gitignore:36]: just keep it, so we don't try to add it again

**Pull request merged**